dev/config/keycloak/import/ltt-realm.json

```diff
@@ -136,7 +136,7 @@
        "alwaysDisplayInConsole": false,
        "clientAuthenticatorType": "client-secret",
        "redirectUris": [
-         "/share"
+         "${KEYCLOAK_SHL_CREATOR_BASE}/share"
```

Comment:
The changes in this PR looks like many fixes to the same typo (thank you), explicitly setting ssl to false for the smtp server (line 815), and this change. 

What is the reason for this change, @victorlima02 ?

Reply:
> Besides the duplicated and corrections. This sets the relative path that can be interpreted into several values depending on its usage, root configuração and context where it is used, this new settings remove ambiguity. About the SSL, this will not disabled secure channels as starttls is set to true. CIRG mail servers have consistently responded to starttls but applications that use SSL directly might fail depending on its library version and options, by setting to false it defaults to starttls protocol which is always well behaved in CIRG mail servers, and the application will have no option besides using starttls, avoiding obscure bugs across servers, versions and options. Last month we had an issue with the mail server and this is the end fix for the client that had a change in behavior due to version change.
> […](#)
> On Thu, Jun 13, 2024, 18:45 Justin McReynolds ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In dev/config/keycloak/import/ltt-realm.json <[#36 (comment)](https://github.com/uwcirg/ltt-environments/pull/36#discussion_r1638983537)> : > @@ -136,7 +136,7 @@ "alwaysDisplayInConsole": false, "clientAuthenticatorType": "client-secret", "redirectUris": [ - "/share" + "${KEYCLOAK_SHL_CREATOR_BASE}/share" The changes in this PR looks like many fixes to the same typo (thank you), explicitly setting ssl to false for the smtp server (line 815), and this change. What is the reason for this change, @victorlima02 <https://github.com/victorlima02> ? — Reply to this email directly, view it on GitHub <[#36 (review)](https://github.com/uwcirg/ltt-environments/pull/36#pullrequestreview-2117085781)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABINEGVC6VRWVLKQAE4AQF3ZHIHIXAVCNFSM6AAAAABJJCL2OSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCMJXGA4DKNZYGE> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Thanks for explaining that Victor. I'll defer to @daniellrgn on this, since the shl client is his.

Reply:
Thanks for the explanation @victorlima02 !